content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -76,32 +76,63 @@ how to [authenticate apiservers](#authenticate-apiservers).
 ### Deploy the admission webhook service
 
 The webhook server in the e2e test is deployed in the Kubernetes cluster, via
-the [deployment API](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#deployment-v1beta1-apps).
+the [deployment API](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#deployment-v1-apps).
 The test also creates a [service](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#service-v1-core)
 as the front-end of the webhook server. See
-[code](https://github.com/kubernetes/kubernetes/blob/v1.13.0/test/e2e/apimachinery/webhook.go#L227).
+[code](https://github.com/kubernetes/kubernetes/blob/v1.15.0/test/e2e/apimachinery/webhook.go#L301).
 
 You may also deploy your webhooks outside of the cluster. You will need to update
-your [webhook client configurations](https://github.com/kubernetes/kubernetes/blob/v1.13.0/staging/src/k8s.io/api/admissionregistration/v1beta1/types.go#L247) accordingly.
+your webhook configurations accordingly.
 
 ### Configure admission webhooks on the fly
 
 You can dynamically configure what resources are subject to what admission
 webhooks via
-[ValidatingWebhookConfiguration](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#validatingwebhookconfiguration-v1beta1-admissionregistration-k8s-io)
+[ValidatingWebhookConfiguration](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#validatingwebhookconfiguration-v1-admissionregistration-k8s-io)
 or
-[MutatingWebhookConfiguration](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#mutatingwebhookconfiguration-v1beta1-admissionregistration-k8s-io).
+[MutatingWebhookConfiguration](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#mutatingwebhookconfiguration-v1-admissionregistration-k8s-io).
 
 The following is an example `validatingWebhookConfiguration`, a mutating webhook
 configuration is similar. See the [webhook configuration](#webhook-configuration) section for details about each config field.
 
+{{< tabs name="ValidatingWebhookConfiguration_example_1" >}}
+{{% tab name="admissionregistration.k8s.io/v1" %}}
+```yaml
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: "<name of this configuration object>"
+webhooks:
+- name: "<webhook name, e.g., pod-policy.example.io>"
+  rules:
+  - apiGroups:
+    - ""
+    apiVersions:
+    - v1
+    operations:
+    - CREATE
+    resources:
+    - pods
+    scope: "Namespaced"
+  clientConfig:
+    service:
+      namespace: "<namespace of the front-end service>"
+      name: "<name of the front-end service>"
+    caBundle: "<pem encoded ca cert that signs the server cert used by the webhook>"
+  admissionReviewVersions:
+  - v1
+  - v1beta1
+  timeoutSeconds: 5
+```
+{{% /tab %}}
+{{% tab name="admissionregistration.k8s.io/v1beta1" %}}
 ```yaml
```

Comment:
How about a comment in the YAML to highlight that this is a beta / superseded API version?

(and something similar elsewhere)

Reply:
the v1 example is the default, and is right next to the v1beta1 version everywhere. do you still think we need to mention it in the v1beta1 tab body?

Reply:
A nice-to-have.
I think it'd help in case the example is copied into other docs.

Reply:
ok, updated